Makefile

```diff
@@ -1,53 +1,78 @@
-.PHONY: clean test dist test-dist
+PROJ := portable8439
 
-src = $(wildcard src/*.c) $(wildcard src/chacha-portable/*.c) $(wildcard src/poly1305-donna/*.c)
+STD ?= -std=c99
+CFLAGS ?= -O3
+CFLAGS += $(STD) -Wpedantic -Wall -Wextra -Isrc -Isrc/chacha-portable -Isrc/poly1305-donna -fstack-protector
+LDFLAGS :=
+VERSION ?= dev-version
+PREFIX ?= /usr/local
 
-CFLAGS?=-O3
-CFLAGS+=-std=c99 -pedantic -Wall -Wextra -Isrc -Isrc/chacha-portable -Isrc/poly1305-donna -fstack-protector
-LDFLAGS = 
+SRCDIR := src
+BLDDIR := dist
+OBJDIR := $(BLDDIR)/obj
+TSTDIR := $(BLDDIR)/test
 
-VERSION?=dev-version
+SOURCES := $(shell find $(SRCDIR) -type f -iname '*.c')
```

Comment:
Why is the `wildcard` feature of make not used? I also run the code on windows, and there find is not present.

Reply:
I don't have an easy work-around for this actually. Make's `wildcard` doesn't support recursive matching (and I'm relying on being able to find all source files so that the tests can be built in a uniform manner).

Reply:
okay, just wanted to make sure there is not a simpler fix